keyboards/hexon38/keymaps/default/keymap.c

```diff
@@ -0,0 +1,426 @@
+// see https://github.com/pepaslabs/hexon38
+
+#include "hexon38.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```